Libraries/src/Amazon.Lambda.AspNetCoreServer.Hosting/ServiceCollectionExtensions.cs

```diff
@@ -88,6 +88,73 @@ public static IServiceCollection AddAWSLambdaHosting(this IServiceCollection ser
             return services;
         }
 
+        #if NET8_0_OR_GREATER
+        /// <summary>
+        /// Adds a function meant to initialize the asp.net and lambda pipelines during <see cref="SnapshotRestore.RegisterBeforeSnapshot"/>
+        /// improving the performance gains offered by SnapStart.
+        /// <para />
+        /// Pass a function with one or more <see cref="HttpRequestMessage"/>s that will be used to invoke
+        /// Routes in your lambda function.  The returned <see cref="HttpRequestMessage"/> must have a relative
+        /// <see cref="HttpRequestMessage.RequestUri"/>.
+        /// <para />.
+        /// Be aware that this will invoke your applications function handler code
```

Comment:
Give the reason that function will be called multiple times which is to influence .NET JIT that this code is a hot path and should be optimized.

Reply:
wordsmithed